<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownLoadDialog</class>
 <widget class="QDialog" name="DownLoadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lb_filename">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>文件名:---</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_filesize">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>文件大小:---</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="probar_download">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="lb_process">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下载进度:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_speed">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前速度:--- KB/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_time">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>预计时间  00:00:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_pause">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_close">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_openposition">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
